### DISCUSSION TITLE
Custom Blog Page

### DISCUSSION THREAD

**Discussion by ExeedFit** [OrchardCMS/OrchardCore#8917] [General]
I am new but trying to implement orchard core. Want to build a custom Blog Summary page. Something very similar to
https://demo.tagdiv.com/newsmag_classic_blog/
Is there an example which I can take, I am trying to use the blog theme but having hard time of getting handle/access to blog ListPart to show the list of popular blogs or all categories and tags to show on right bar.

Is there an example I can start with

**Comment by Skrypt** [comment]
List of popular blogs would be a custom Query.
Categories and tags would be Taxonomies.

**Reply by ExeedFit** [reply]
Thanks for sharing. However, my question is specifically how to get access to list of all Categories, Taxonomy  and blogs while on content_BlogPost.liquid

I tried using the following code to get access to Blog but it only gives me access to Blog but not the list of Blog-Posts

{% assign my_content4 = Content.Latest["slug:blog"] %}

How do I get access to the list part which has list of all the BlogsPost ?

Is there a way to short based on how many views each blog post has, so I can sort it by most popular?
